src/shared/modules/discovery/discoveryDuck.ts

```diff
@@ -158,10 +158,7 @@ export const discoveryOnStartupEpic = (some$: any, store: any) => {
         action.requestedUseDb = passedDb
       }
 
-      const discoveryURL =
-        searchParams.get('discoveryURL') ||
-        searchParams.get('discoveryurl') ||
-        searchParams.get('discoveryUrl')
+      const discoveryURL = searchParams.get('discoveryURL')
```

Comment:
It might be nice to have all these search param keys be constants defined in a single place?

Reply:
I agree, but I think refactoring to get all url params in the same place is out of scope for the SSO work